EveripediaNetwork/ep-ui
# Review Event Page

# Review Event Page

This PR works on 
- Hiding the Nearby event feature on the page from the public while we begin to fix the issue and test it properly.
- Sorting the events on the homepage by most recent for instance the recent event should be in August 2024 by properly sorting
- Archiving Old events

## How to test
- Go to events page, check if the events are starting from the current date upward
- Try testing out all filters to see if all they start from the current date upwards
- Confirm if nearbyEvent widget is hidden on events and events/:id
- Open any wiki, create a new event metadata, update and also remove. Confirm if it works fine
- do the same for an event wiki

## Linked issues

closes https://github.com/EveripediaNetwork/issues/issues/3016
